src/server/saved_objects/service/lib/repository.js

```diff
@@ -606,4 +614,10 @@ export class SavedObjectsRepository {
     const savedObject = this._serializer.rawToSavedObject(raw);
     return omit(savedObject, 'namespace');
   }
+
+  assertAllowedType(type) {
```

Comment:
Everyone knows that `type` is a terrible name for this variable because it can be either a single type or an array of types, this assertion needs to handle both.

Reply:
Ok updated to handle single or array of strings.